web/kratos/.env.example

```diff
@@ -1,3 +1,3 @@
 KRATOS_URL=http://172.17.0.1:8000/kratos
-KRATOS_ADMIN_URL=http://172.17.0.1:9434
-CRUX_UI_URL=http://172.17.0.1:8000
\ No newline at end of file
+KRATOS_ADMIN_URL=http://172.17.0.1:4434
+CRUX_UI_URL=http://172.17.0.1:8000
```

Comment:
please use localhost as host, since ports are exposed using CLI, and with the exception of Docker+Linux these IP addresses won't work.